manifest/default.xml

```diff
@@ -23,7 +23,7 @@
 
   <!-- Build Scripts (required on CI servers) -->
   <project name="build" path="cbbuild" remote="couchbase">
-    <annotation name="VERSION" value="1.5.0"     keep="true"/>
+    <annotation name="VERSION" value="1.5.1"     keep="true"/>
```

Comment:
@ceejatec is this the only thing needed to have the master branch SG builds named 1.5.1-x?  I didn't see any version info under the product-config.json for default.xml

Reply:
Yes, that should be it for master, I believe.

Reply:
@ceejatec We noticed that the latest master build was 1.5.1-587 -- is there any way to reset the counter for master builds?